pandas/tests/frame/indexing/test_where.py

```diff
@@ -499,6 +501,9 @@ def test_where_axis(self):
         assert return_value is None
         tm.assert_frame_equal(result, expected)
 
+    # TODO(ArrayManager) rewrite test to be valid
+    @td.skip_array_manager_invalid_test
```

Comment:
Note: I removed those skips in this file, as the tests are now passing. But for some reason github doesn't yet pick up the last commit I pushed